validation/Validation.h

```diff
@@ -28,9 +28,17 @@
 #include "../autogtp/Game.h"
 #include "Results.h"
 
+typedef struct engine {
+    QString binary;
+    QString options;
+    QString network;
+    QStringList commands;
+} engine_t;
+
```

Comment:
I would make a proper Object with copy constructor, maybe even a move constructor and call it Engine. And maybe adding a member to handle the list of command.

Reply:
Do you have a preference between using a struct or a class?
What do you mean by "handle the list of command"? I'm not understanding what I could add that would be helpful.